.github/workflows/ci.yml

```diff
@@ -26,6 +26,9 @@ jobs:
         include:
           - os: ubuntu-latest
             run_supabase_tests: true
+          - os: ubuntu-latest
+            run_supabase_tests: false
```

Comment:
Why is this necessary?

Reply:
We are choosing to set the flag for the existing combination ubuntu-latest/false.